ccmbioinfo/lfs
# CARDS-2261: PREMS: Most emergency patients are being discarded during import

To test:
- build with `-Pdocker`
- in compose-cluster run `python3 generate_compose_yaml.py --mssql --cards_project cards4prems --oak_filesystem --dev_docker_image --composum --adminer`
- run `docker-compose build && docker-compose up`
- at `http://localhost:1435/?mssql=mssql&username=sa&db=master&ns=path&import=` import `compose-cluster/mssql/processorImportTestSql.sql`
- open `http://localhost:8080/Subjects.importClarity`
- wait a few seconds for the import to finish
- check that patients 36, 37, and 39 were imported
- check that patients 34, 35, and 38 were not imported 
- shut down and run `docker-compose rm -vf && ./cleanup.sh` to clean up everything
